app/views/posts/show.html.erb

```diff
@@ -13,13 +13,12 @@
       </div>
       <%= link_to @post.user.username, user_path(@post.user), class: 'username' %>
     </div>
-    <div class="description">
-      <%= render partial: "description", locals: {post: @post} %>
-    </div>
+    <%= @post.description %>
     <div class="delete">
       <% if current_user.posts.exists?(@post.id) %>
```

Comment:
this line should not be there in view, views should be as dumb as possible. This line queries to db to check .